### PR TITLE
Common challenges

### PR DESCRIPTION
**Pull request by ChristinaLK**
notes from 2021-01-05 community discussion

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -33,6 +33,20 @@ If you've written up a diagram of the data analysis pipeline (raw data ->
...
 helpful to identify that you're now somewhere between clean data and analysis.
 
+## Common Difficult Concepts
+
+* Making the leap from a research question to a query (seen in some of the challenges)
+* Data import
+    * Not necessarily a concept, but we always have at least a handful of people that 
+      struggle with the table import step -- especially in the virtual context with window changing
+    * Data type options in SQLite (Integer, text, blog, real, numeric) when importing 
+      from CSV. (Maybe have a table of SQLite date types in the student matieral).
```
Comment (nsoranzo) [inline _extras/guide.md:43]:
```suggestion
      from CSV. (Maybe have a table of SQLite date types in the student mateiral).
```

File: _extras/guide.md [hunk 2]
```diff
@@ -33,6 +33,20 @@ If you've written up a diagram of the data analysis pipeline (raw data ->
...
+* Good to make sure that a comparison is drawn between joins in different 
+languages, e.g. SQL vs tidyverse
+* HAVING, and why it's different to WHERE.... - especially when teaching online
+* given the dataset is _relatively_ complex, some students (and instructors) find it difficult to remember what data is where, especially when they're from a totally different domain 
+* How nulls behave in different circumstances (when did a NULL change your result and how do you know?)
```
Comment (nsoranzo) [inline _extras/guide.md:48]:
```suggestion
* How NULLs behave in different circumstances (when did a NULL change your result and how do you know?)
```

File: _extras/guide.md [hunk 3]
```diff
@@ -33,6 +33,20 @@ If you've written up a diagram of the data analysis pipeline (raw data ->
...
 helpful to identify that you're now somewhere between clean data and analysis.
 
+## Common Difficult Concepts
+
+* Making the leap from a research question to a query (seen in some of the challenges)
+* Data import
+    * Not necessarily a concept, but we always have at least a handful of people that 
+      struggle with the table import step -- especially in the virtual context with window changing
+    * Data type options in SQLite (Integer, text, blog, real, numeric) when importing 
+      from CSV. (Maybe have a table of SQLite date types in the student mateiral).
```
Comment (nsoranzo) [inline _extras/guide.md:43]: It seems my previous suggestion was only a partial improvement :laughing: 
```suggestion
      from CSV. (Maybe have a table of SQLite date types in the student material).
```

Comment (ChristinaLK) [reply _extras/guide.md:43]: ha! and I thought I read it carefully. 🤪

File: _extras/guide.md [hunk 4]
```diff
@@ -33,6 +33,20 @@ If you've written up a diagram of the data analysis pipeline (raw data ->
 clean data -> import and analyze -> results -> visualization), it can be
 helpful to identify that you're now somewhere between clean data and analysis.
 
+## Common Difficult Concepts
+
+* Making the leap from a research question to a query (seen in some of the challenges)
+* Data import
+    * Not necessarily a concept, but we always have at least a handful of people that 
+      struggle with the table import step -- especially in the virtual context with window changing
+    * Data type options in SQLite (Integer, text, blog, real, numeric) when importing
```
Comment (nsoranzo) [inline _extras/guide.md:42]:
```suggestion
    * Data type options in SQLite (Integer, text, blob, real, numeric) when importing 
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**